### PR TITLE
Multi-language support

### LINKED ISSUE #66: Multi-Language Support
- Need language selector with predefined list of languages
- Language string (en, zh, fr, etc) needs to be added to the URLs (URLs should be dynamically added to BTR over using the `.dojorc` file).
- The selected language should be passed to the blocks.

Relates to #53

### PR DESCRIPTION
**Pull request by agubler**
## Description

Adds support for multi language deployments of the site. Starting with `zh-CN`.

Builds on the existing changes from @KaneFreeman here #171 

Resolves #66

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-4m8scgfse.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-8gtl8p69h.now.sh

File: src/footer/Footer.tsx [hunk 1]
```diff
@@ -230,6 +231,10 @@ export default factory(function Footer({ middleware: { theme, i18n } }) {
 									v5.0
 									<img classes={themedCss.externalLink} alt="externalLink" src={externalLink} />
 								</a>
+								<div classes={themedCss.title}>{messages.languages}</div>
+								<a href="https://zh-CN.dojo.io" classes={css.link}>
```
Comment (KaneFreeman) [inline src/footer/Footer.tsx]: We should probably have a link for english here too for when they are on another language's version.

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-cbgq1qkp3.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-6n16y69h6.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-mx2rn875k.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-8b8eltpve.now.sh

**Pull request merged**